tests/src/serverTestMultiMappedPV.cpp

```diff
@@ -0,0 +1,31 @@
+#define BOOST_TEST_MODULE serverTestMultiMappedPV
+#include <boost/test/included/unit_test.hpp>
+
+#include "DoocsAdapter.h"
+#include "serverBasedTestTools.h"
+#include <ChimeraTK/ControlSystemAdapter/Testing/ReferenceTestApplication.h>
+#include <doocs-server-test-helper/doocsServerTestHelper.h>
+
+extern const char* object_name;
+#include <doocs-server-test-helper/ThreadedDoocsServer.h>
+
+using namespace boost::unit_test_framework;
+using namespace boost::unit_test;
+using namespace ChimeraTK;
+
+DOOCS_ADAPTER_DEFAULT_FIXTURE_STATIC_APPLICATION
+
+/// Check that all expected variables are there.
+BOOST_AUTO_TEST_CASE(testMultiMapped, *boost::unit_test::tolerance(0.0001)) {
+  GlobalFixture::referenceTestApplication.runMainLoopOnce();
+
+  TEST_WITH_TIMEOUT(DoocsServerTestHelper::doocsGet<float>("//FIRST/SCALAR_1") == 12.345);
+  TEST_WITH_TIMEOUT(DoocsServerTestHelper::doocsGet<float>("//SECOND/SCALAR_1") == 12.345);
+
+  DoocsServerTestHelper::doocsSet<float>("//FIRST/SCALAR_1", 42.42);
+
+  GlobalFixture::referenceTestApplication.runMainLoopOnce();
```

Comment:
I think it should work without triggering the application loop. Some queues might run over, though.

Reply:
right, removed.